### PR TITLE
[Pages] mailchannels review

### PR DESCRIPTION
**Pull request by deadlypants1973**
closes #12057

### CODE REVIEW

File: content/pages/functions/plugins/mailchannels.md [hunk 1]
```diff
@@ -29,12 +31,13 @@ import mailChannelsPlugin from "@cloudflare/pages-plugin-mailchannels";
...
   personalizations: [
     {
-      to: [{ name: "ACME Support", email: "support@example.com" }],
+      to: [{ name: "Some User", email: "someuser@example.com" }],
     },
   ],
   from: {
     name: "ACME Support",
-    email: "support@example.com",
+    // The domain of your `from` address must be the same as the domain you setup MailChannels Domain Lockdown for (detailed below)
```
Comment (pedrosousa) [inline content/pages/functions/plugins/mailchannels.md]:
```suggestion
    // The domain of your `from` address must be the same as the domain you set up MailChannels Domain Lockdown for (detailed below)
```

File: content/pages/functions/plugins/mailchannels.md [hunk 2]
```diff
@@ -125,28 +164,35 @@ $ echo -n "v=DKIM1;p=" > dkim_record.txt && openssl rsa -in private_key.pem -pub
...
+| Type | Name                      | Content                                  |
+|------|---------------------------|------------------------------------------|
+| TXT  | `mailchannels._domainkey` | Insert the contents of `dkim_record.txt` | 
+
+{{</example>}}
 
-{{<Aside type= "note" header="Selector value">}}
-You can choose any value as the selector, as long as it is permitted as a DNS hostname (that is, all lowercase letters, numbers and hyphens).
+{{<Aside type= "note" header="Use a different selector">}}
+You can substitute `mailchannels.domainkey` for any name with the format `<selector_key>._domainkey`. You can choose any value as the selector, as long as it is permitted as a DNS hostname (that is, all lowercase letters, numbers and hyphens).
```
Comment (pedrosousa) [inline content/pages/functions/plugins/mailchannels.md]:
```suggestion
You can substitute `mailchannels._domainkey` for any name with the format `<selector_key>._domainkey`. You can choose any value as the selector, as long as it is permitted as a DNS hostname (that is, all lowercase letters, numbers, and hyphens).
```

File: content/pages/functions/plugins/mailchannels.md [hunk 3]
```diff
@@ -155,22 +201,26 @@ The following code block shows an example of using DKIM with the MailChannels Pa
...
 import mailChannelsPlugin from "@cloudflare/pages-plugin-mailchannels";
 
 export const onRequest: PagesFunction = (context) => mailChannelsPlugin({
   personalizations: [
     {
-      to: [{ name: "Some User", email: "user@cloudflare.com" }],
-      "dkim_domain": "example.com", // The value has to be the domain you added DKIM records to and where you're sending your email from
-      "dkim_selector": "mailchannels",
+      to: [{ name: "Some User", email: "someuser@example.com" }],
+      // This value has to be the domain you added DKIM records to and where you're sending your email from
```
Comment (pedrosousa) [inline content/pages/functions/plugins/mailchannels.md]:
```suggestion
      // This value has to be the domain you added DKIM records to and where you are sending your email from
```

File: content/pages/functions/plugins/mailchannels.md [hunk 4]
```diff
@@ -155,22 +201,26 @@ The following code block shows an example of using DKIM with the MailChannels Pa
...
+      "dkim_domain": "mydomain.com", 
+      // This value has be the same as the selector you chose for your DKIM record name
+      // For example, use "mailchannels" if you used mailchannels._domainkey as your record name
```
Comment (pedrosousa) [inline content/pages/functions/plugins/mailchannels.md]:
```suggestion
      // For example, use "mailchannels" if you used "mailchannels._domainkey" as your record name
```

File: content/pages/functions/plugins/mailchannels.md [hunk 5]
```diff
@@ -155,22 +201,26 @@ The following code block shows an example of using DKIM with the MailChannels Pa
...
+      "dkim_selector": "mailchannels", 
       "dkim_private_key": context.env.DKIM_PRIVATE_KEY
     },
   ],
   from: {
     name: "ACME Support",
-    email: "support@example.com",
+    // The domain of your `from` address must be the same as the domain you setup MailChannels Domain Lockdown for
```
Comment (pedrosousa) [inline content/pages/functions/plugins/mailchannels.md]:
```suggestion
    // The domain of your `from` address must be the same as the domain you set up MailChannels Domain Lockdown for
```

**Review by pedrosousa — APPROVED** [review]

File: content/pages/functions/plugins/mailchannels.md [hunk 6]
```diff
@@ -74,18 +77,53 @@ The `method` and `action` attributes of the HTML form do not need to be set. The
 
 For more information about MailChannels and the options they support, refer to [the documentation](https://mailchannels.zendesk.com/hc/en-us/articles/4565898358413-Sending-Email-from-Cloudflare-Workers-using-MailChannels-Send-API).
 
+## Enable MailChannels for your account - Domain Lockdown
+
+To enable MailChannels to send emails on your behalf through Cloudflare Pages Functions, you have to setup a [Domain Lockdown DNS record](https://support.mailchannels.com/hc/en-us/articles/16918954360845-Secure-your-domain-name-against-spoofing-with-Domain-Lockdown-) for the domain you are sending emails from.
+
+For most cases, setting up a Domain Lockdown DNS record involves adding a `TXT` entry to your DNS. Refer to the following steps to set up a Domain Lockdown DNS record.
+
+First, find your account's `workers.dev` subdomain:
```
Comment (hekuli) [inline content/pages/functions/plugins/mailchannels.md:86]: I believe there is an error in this portion of the docs. This did not work for me. I kept getting email send error messages. 

What did work however was using the value from the pages application:
`Whatever.pages.dev`
and my custom domain `mydomain.com`

I am not the only one, see:
https://stackoverflow.com/a/77041550/275016

A lot of people on the internet are confused and having problems. Please re-evalute this part of the docs 🙏

**Pull request merged**